openjdk.test.mauve/build.xml

```diff
@@ -360,16 +360,35 @@ limitations under the License.
 	</target>
 
 	<target name="get-source" depends="download-cvsclient, create-work-dir" unless="openjdk_test_mauve_already_built">
-		<exec dir="${openjdk_test_mauve_work_dir}" executable="java">
-		<arg value="-Dcvs.root=:pserver:anoncvs@sourceware.org:/cvs/mauve"/>
-		<arg value="-jar"/>
-		<arg value="${first_prereqs_root}/cvsclient/org-netbeans-lib-cvsclient.jar"/>
-		<arg value="export"/>
-		<arg value="-r"/>
-		<arg value="HEAD"/>
-		<arg value="mauve"/>
-</exec>
-	</target>
+         <if>
```

Comment:
In aqa-systemtest Ant build scripts, we have made a conscious effort to not use anything that introduces a dependency on ant-contrib. `<if>` is an ant-contrib task. Hence, the "if" condition here should ideally be re-written. For example, something like the following would suffice: 

```
<condition property="isJavaHomeSet" value="true">
	<isset property="JAVA_HOME"/>
</condition>

<condition property="isTestJDKHomeSet" value="true">
	<isset property="TEST_JDK_HOME"/>
</condition>

<target name="get-source-using-java-home" if="isJavaHomeSet".........> 
    copy-paste the details of the exec command with java...
</target>

<target name="get-source-using-test-jdk-home" if="isTestJDKHomeSet".........> 
   copy-paste the details of the exec command with TEST_JDK_HOME/bin/java>
</target>

<target name="get-source" depends="download-cvsclient, create-work-dir" unless="openjdk_test_mauve_already_built">
  <antcall target="get-source-using-java-home"/>
  <antcall target="get-source-using-test-jdk-home"/>
</target>
```

Reply:
Ggg